target/linux/ath79/patches-5.15/0040-MIPS-ath79-common-exports.patch

```diff
@@ -0,0 +1,23 @@
+From: Luiz Angelo Daros de Luca <luizluca@gmail.com>
+Subject: [PATCH] ath79: export ath79_pll_base
+
+This symbol is declared as extern but nobody exported it.
+Any module including arch/mips/include/asm/mach-ath79/ath79.h
+will not build.
```

Comment:
Please add the error message to the patch comment.
````
Without this export ag71xx.ko will not build as a module and the build will fail like this;
ERROR: modpost: "ath79_pll_base" [drivers/net/ethernet/atheros/ag71xx/ag71xx.ko] undefined!
The ath79_pll_base symbol is accessed in the ath79_pll_wr() inline function. 
````

Reply:
done